com.cubrid.common.ui/src/com/cubrid/common/ui/common/preference/JdbcManageComposite.java

```diff
@@ -139,7 +140,32 @@ public void save() {
 	private void createJdbcTableGroup(Composite composite) {
 		final String[] columnNameArr = new String[]{
 				Messages.tblColDriverVersion, Messages.tblColJarPath };
+		
 		TableViewerSorter sorter = new TableViewerSorter();
+		sorter.setColumnComparator(0, new Comparator<Object>(){
+			public int compare(Object o1, Object o2){
+				if(o1 instanceof String && o2 instanceof String){
+					String s1 = (String)o1;
+					String s2 = (String)o2;
+				
+					String[] version1Tokens = s1.substring(s1.lastIndexOf('-')+1).split("\\.");
+					String[] version2Tokens = s2.substring(s2.lastIndexOf('-')+1).split("\\.");
+					
+					int size = Math.min(version1Tokens.length, version2Tokens.length);
+					
+					for(int i = 0; i < size; i++){
+						Integer first = Integer.parseInt(version1Tokens[i]);
+						Integer second = Integer.parseInt(version2Tokens[i]);
+						if(first != second){
+							return second-first;
```

Comment:
Actually, there are left something indicator what is current ordering in the first column of the tableview like this.

<img width="179" alt="si" src="https://cloud.githubusercontent.com/assets/1791415/20931902/0a1c9098-bc16-11e6-9c7b-db0b5dc81fdf.png">

If you add the following code under the following line of code, you can see the sorting indicator in the first column.

```
jdbcInfoTv.getTable().setSortColumn(jdbcInfoTv.getTable().getColumn(0));
jdbcInfoTv.getTable().setSortDirection(sorter.isAsc() ? SWT.UP : SWT.DOWN);
```

https://github.com/CUBRID/cubrid-manager/pull/21/files#diff-03909d2171ae623ee5fa9dba900243beR178